packages/docusaurus-theme-classic/src/theme/DocCard/index.tsx

```diff
@@ -70,7 +71,15 @@ function CardCategory({item}: {item: PropSidebarItemCategory}): JSX.Element {
       href={href}
       icon="🗃️"
```

Comment:
Interesting idea, is it possible that we allow customization of these icons through translations? I still feel it's valuable to allow doing so without swizzling.

Reply:
I'm not 100% convinced that we should add emoji icons that way, it was just a quick thing to ship the new feature that we can improve later.

IMHO we should make swizzle a real first-class citizen and allow customizing all site icons in a similar way, even if the react component can eventually just render an emoji